assets/model_monitoring/components/src/action_analyzer/action_analyzer_identify_problem_traffic/run.py

```diff
@@ -89,10 +89,10 @@ def bertopic_get_topic(queries,
                          azure_endpoint=azure_endpoint_domain_name,
                          azure_deployment=model_deployment_name)
     representation_model = OpenAI(client, model=model_deployment_name, chat=True, prompt=BERTOPIC_DEFAULT_PROMPT)
-    topic_model = BERTopic(
-        min_topic_size=3,
+    topic_model = BERTopic( # [fep] is there way to also control the max number of clusters?
```

Comment:
I did not find in the official doc. Can discuss offline

https://maartengr.github.io/BERTopic/getting_started/parameter%20tuning/parametertuning.html#language